### PR TITLE
Update github pr template and add hacking.md file

### LINKED ISSUE #722: Update GitHub PR template

### PR DESCRIPTION
**Pull request by solazio**
## Done
- _Update GH pr template_
- _Add HACKING.md file_
- _Update Readme.md file_

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card
Fixes #722

## Screenshots
[if relevant, include a screenshot]

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Status checks and prometheus metrics
+
+[Talisker](https://talisker.readthedocs.io/en/latest/) provides a bunch of useful status checks and metrics about the running application. Some of this information is sensitive and so to access it you need to run the site with your IP address mentioned in the `TALISKER_NETWORKS` variable.
+
+Now visit http://127.0.0.1:8004/_status to see the endpoints provided by Talisker. Useful ones include:
+
+- http://127.0.0.1:8004/_status/check - A basic check that the site is running
+- http://127.0.0.1:8004/_status/metrics - The prometheus metrics for the application
```
Comment (jkfran) [inline HACKING.md:45]: Update the port here to `8045`

Comment (solazio) [reply HACKING.md]: good catch. I missed it.

**Review by jkfran — APPROVED** [review]
Looks good to me!

File: HACKING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Using Sentry error tracker
+
+For development purposes, visit https://sentry.io/signup/, signup and setup a project. By then you will have a sentry DSN string like:
+
+```
+https://<user>:<secret>@sentry.io/<project_id>
+```
+
+Create or update you `.env.local` file:
```
Comment (pmahnke) [inline HACKING.md:22]: 'your' not 'you'

Comment (solazio) [reply HACKING.md:22]: thanks, will update it in a separate pr as this is merged already

**Pull request merged**